app/helpers/record_helper.rb

```diff
@@ -61,4 +61,18 @@ def script_class(field)
   def script_text_class(field)
     field[:script] && current_user.arc? ? "script_text" : "no_script_text"
   end
+
+  def record_is_in_progress(record)
```

Comment:
if this is returning a boolean, it might be better to have the method name have a question mark - maybe like `def record_in_progress?(record)`

Reply:
it even sounds like it needs a question mark

Reply:
Always forget about this little '?' nuance, thanks.